src/components/bookmarks-panel/slider.tsx

```diff
@@ -46,13 +46,19 @@ const ITEM_GAP = 16;
 type SliderProps = {
   bookmarks: Bookmark[];
   editingMode: boolean;
+  editingBookmarkId: string;
   onSelectBookmark: (id: string) => void;
+  onDeleteBookmark: (id: string) => void;
+  onEditingBookmark: (id: string) => void;
 };
 
 export const Slider = ({
   bookmarks,
   editingMode,
+  editingBookmarkId,
   onSelectBookmark,
+  onDeleteBookmark,
+  onEditingBookmark
 }: SliderProps) => {
   const [position, setPosition] = useState<number>(0);
   const [selectedBookmark, setSelectedBookmark] = useState<null | Bookmark>(
```

Comment:
Are editingBookmark and selectedBookmark same item? Could we move selectedBookmark to parent component